packages/docs/components/button.md

```diff
@@ -25,101 +25,83 @@ links:
 
 ## Anatomy
 
```

Comment:
we usually want a `Description` to follow a section heading, it adds additional spacing between the content and the imagery. I noticed you haven't been doing this, is there a reason? If so let me know, and we can figure out a path forward here -- the current state is visually broken imo

Reply:
We're pulling all of the copy out of the Anatomy sections, so we'll need to solve it at the Heading level probably.

IMO `<Description>` shouldn't be required for proper spacing here. That is, we should be able to include an image under a heading without bespoke adjustments. Feels like one more thing to inventory re: Type.

@joejezowski-okta any thoughts here?

Reply:
I agree with @arnoldsandoval-okta here. If we don't have copy, we need some extra space for breathing room.

Reply:
Mk, do y'all mind tackling this separately, since it'll apply to all of our Component docs now?

Let's make sure to add this one to the Type ticket too.

Reply:
update: I've included a fix for this here, in `_shame.scss`, instead of deferring.